README.md

```diff
@@ -1,44 +1,99 @@
-<div align="center" style="display:flex;flex-direction:column;"}>
-  <a href="https://excalidraw.com">
-    <img width="540" src="./public/og-image-sm.png" alt="Excalidraw logo: Sketch handrawn like diagrams."/>
-  </a>
-  <h3>Virtual whiteboard for sketching hand-drawn like diagrams.<br/>Collaborative and end-to-end encrypted.</h3>
-  <p>
-    <a href="https://twitter.com/excalidraw">
-      <img alt="Follow Excalidraw on Twitter" src="https://img.shields.io/twitter/follow/excalidraw.svg?label=follow+@excalidraw&style=social&logo=twitter"/>
-    </a>
-    <a href="https://discord.gg/UexuTaE">
-      <img alt="Chat with us on Discord" src="https://img.shields.io/discord/723672430744174682?color=738ad6&label=Chat%20on%20Discord&logo=discord&logoColor=ffffff&widge=false"/>
-    </a>
-  </p>
-</div>
+<a href="https://excalidraw.com/" target="_blank" rel="noopener">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" alt="Excalidraw" srcset="https://excalidraw.nyc3.cdn.digitaloceanspaces.com/github%2FExcalidraw_Github_cover_dark.png" />
+    <img alt="Excalidraw" src="https://excalidraw.nyc3.cdn.digitaloceanspaces.com/github%2FExcalidraw_Github_cover.png" />
+  </picture>
+</a>
 
-## Try now
+<h4 align="center">
+  <a href="https://blog.excalidraw.com">Blog</a> |
+  <a href="https://docs.excalidraw.com">Documentation</a> |
+  <a href="https://excalidraw.com">Excalidraw Editor</a> |
+  <a href="https://plus.excalidraw.com">Excalidraw+</a>
```

Comment:
swap order in this way?

```
Excalidraw Editor
Documentation
Blog
Excalidraw+
```

Reply:
try it - not visually appealing

Reply:
went with:

![image](https://user-images.githubusercontent.com/5153846/218862856-c8cb7e77-83bc-4161-9b4a-d12cdebf8f78.png)

We have to have the editor as first link